hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/Driver.java

```diff
@@ -55,7 +56,8 @@ public static void main(String[] args) throws Throwable {
       "Export" + " the specific snapshot to a given FileSystem.");
     pgd.addClass(MobRefReporter.NAME, MobRefReporter.class, "Check the mob cells in a particular "
       + "table and cf and confirm that the files they point to are correct.");
-
+    pgd.addClass(ReplicationPeerMigrationTool.NAME, ReplicationPeerMigrationTool.class,
```

Comment:
@Apache9  sir.
Driver class is in the mapreduce package, and the tools in this class will start mapreduce tasks. 
I am not sure if migratereppeer tool is suitable to be placed in this class.